pinecone/client_test.go

```diff
@@ -24,218 +24,12 @@ import (
 )
 
 // Integration tests:
-func (ts *IntegrationTests) TestNewClientParamsSet() {
```

Comment:
These were all unit tests since they're not actually making any network calls. Moved to the bottom of the file and out of `IntegrationTests`.